index.bs

```diff
@@ -1224,52 +1224,80 @@ as the *sender* and the agent receiving the media as the *receiver*.
 
 Capabilities {#streaming-capabilities}
 --------------------------------------------
+If the advertiser is already authenticated, the requester has the ability to
+request additional information by sending an streaming-capabilities-request
+message, and receive back a streaming-capabilities-response message with the
+following properties:
 
-<!--
-  TODO:
+: receive-audio (required)
+:: A list of audio capabilities. For an explanation of fields, see below.
 
-  Put in something like this:
+: receive-video (required)
+:: A list of video capabilities. For an explanation of fields, see below.
 
-streaming-capabilities-request {
-  request
-}
+: receive-data (required)
+:: A list of arbitrary data formats (represented as codecs) the device supports
+    for receiving data.
```

Comment:
I think I'd rather call codecs formats than call data formats codecs.  It's actually more precise as well, since codecs can be in different profiles/modes.  It's just that we commonly say "codec" instead of "format".

I'll give more explicit suggestions down below.

Reply:
Agreed. I went back and forth internally, this sounds great.